packages/xo-web/src/xo-app/backup/new/mirror/index.js

```diff
@@ -458,6 +461,21 @@ const NewMirrorBackup = decorate([
                           onChange={effects.setHideSuccessfulItems}
                         />
                       </FormGroup>
+                      <FormGroup>
+                        <label htmlFor={state.inputMergeBackupsSynchronously}>
```

Comment:
please add a computed for `inputMergeBackupsSynchronously`.
`inputMergeBackupsSynchronously: generateId`